db/migrate/20200908080357_create_cart_items.rb

```diff
@@ -0,0 +1,10 @@
+class CreateCartItems < ActiveRecord::Migration[5.2]
+  def change
+    create_table :cart_items do |t|
+      t.references :cart, foreign_key: true
+      t.references :item, foreign_key: true
+      t.integer :quantity,default: 0
```

Comment:
Here you are assigning default 0 and while creating object you setting value to 1.
Please re-implement the whole thing. Try to come up with reasonable logic.

Reply:
Yes, that was the mistake and I corrected it.